tests/benchdnn/dnnl_common.hpp

```diff
@@ -969,9 +969,15 @@ void init_memory_args(dnn_mem_map_t &mem_map, const prb_t *prb,
         mem_map.emplace(DNNL_ARG_ATTR_DROPOUT_PROBABILITY,
                 dnn_mem_t(prob_md, test_engine, /* prefill = */ true));
 
-        auto seed_md = dnn_mem_t::init_md(1, &count, dnnl_s32, tag::abx);
+        auto seed_md = dnn_mem_t::init_md(1, &count, dnnl_s64, tag::abx);
         mem_map.emplace(DNNL_ARG_ATTR_DROPOUT_SEED,
                 dnn_mem_t(seed_md, test_engine, /* prefill = */ true));
```

Comment:
Seems like scalar values are not supported yet?

Reply:
added that, thanks!